spk/tvheadend/src/tvheadend.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+HTTPP=$1
+HTSPP=$2
+USRN=`id -un`
+USRG=`id -gn`
+HOME_DIR="${SYNOPKG_PKGDEST}/var"
+LOG_FILE="${HOME_DIR}/tvheadend.log"
+PID_FILE="${HOME_DIR}/tvheadend.pid"
+${SYNOPKG_PKGDEST}/bin/tvheadend -f -u ${USRN} -g ${USRG} --http_port ${HTTPP} --htsp_port ${HTSPP} -c ${HOME_DIR} -l ${LOG_FILE} -p ${PID_FILE}
```

Comment:
I have just tried to use `${GROUP}` instead of `${USRG}` but it is an empty variable when used from within `service-setup.sh`. I will retain `${USRG}` for the time being.

Reply:
You have to hardcode it as one of first lines in `service-setup.sh` like @ymartin59 says a bit up here in the other comments and like I did for SABnzbd: https://github.com/SynoCommunity/spksrc/commit/2ac3e53e73577c7055f3eac688169b43d1cc55fd#diff-5d6bd92e6f38c326c2344d53c1dfb20aR10

Reply:
Ahh, now I see. Misunderstood it. Thanks for clarifying! Will put it in `sc-media` then...

Reply:
Just tried this, but it screws up all the user permissions again. I will revert to the old `tvheadend` goup and force push to get a cleaner version. I also think you have been reviewing outdated commits, as the `tvheadend.sh` file has already been removed some days ago.